_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,155 @@
+nav_order: 5
+has_children: false
+---
+
+# Installing OpenSearch Benchmark
+
+You can install OpenSearch Benchmark directly on a host running Linux or macOS or you can run OpenSearch Benchmark in a Docker container on any compatible host. This page describes high-level considerations for your OpenSearch Benchmark host, as well as instructions for installing OpenSearch Benchmark.
+
+
+## Choosing appropriate hardware
+
+OpenSearch Benchmark can be used to provision OpenSearch nodes for testing. If you intend to use OpenSearch Benchmark to provision nodes in your environment, then install OpenSearch Benchmark directly on each host in the cluster. Additionally, you must configure each host in the cluster for OpenSearch. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/) for guidance on important host settings you should review.
+
+Remember that OpenSearch Benchmark cannot be used to provision OpenSearch nodes when you run OpenSearch Benchmark in a Docker container. If you want to use OpenSearch Benchmark to provision nodes, or if you want to distribute the benchmark workload with the OpenSearch Benchmark daemon, then you must install OpenSearch Benchmark directly on each host using Python and pip.
+{: .important}
+
+When you select a host, you should also think about which workloads you want to run. To see a list of default benchmark workloads, visit the [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) repository on GitHub. As a general rule, make sure that the OpenSearch Benchmark host has enough free storage space to store the compressed data and the fully decompressed data corpus once OpenSearch Benchmark is installed.
+
+If you want to benchmark with a default workload, then use the following table to determine the approximate minimum amount of required free space needed by adding the compressed size with the uncompressed size:
+
+| Workload Name | Document Count | Compressed Size | Uncompressed Size |
+| :----: | :----: | :----: | :----: |
+| eventdata | 20,000,000 | 756.0 MB | 15.3 GB |
+| geonames | 11,396,503 | 252.9 MB | 3.3 GB |
+| geopoint | 60,844,404 | 482.1 MB | 2.3 GB |
+| geopointshape | 60,844,404 | 470.8 MB | 2.6 GB |
+| geoshape | 60,523,283 | 13.4 GB | 45.4 GB |
+| http_logs | 247,249,096 | 1.2 GB | 31.1 GB |
+| nested | 11,203,029 | 663.3 MB | 3.4 GB |
+| noaa | 33,659,481 | 949.4 MB | 9.0 GB |
+| nyc_taxis | 165,346,692 | 4.5 GB | 74.3 GB |
+| percolator | 2,000,000 | 121.1 kB | 104.9 MB |
+| pmc | 574,199 | 5.5 GB | 21.7 GB |
+| so | 36,062,278 | 8.9 GB | 33.1 GB |
+
+Your OpenSearch Benchmark host should use solid-state drives (SSDs) for storage because they perform significantly faster at read and write operations than traditional spinning-disk hard drives. Spinning-disk hard drives can introduce performance bottlenecks, which can make benchmark results unreliable and inconsistent.
```

Comment:
```suggestion
Your OpenSearch Benchmark host should use solid-state drives (SSDs) for storage because they perform read and write operations significantly faster than traditional spinning-disk hard drives. Spinning-disk hard drives can introduce performance bottlenecks, which can make benchmark results unreliable and inconsistent.
```